### ISSUE TITLE
Error: "module 'openai' has no attribute 'ChatCompletion'"

### ISSUE THREAD

**Issue by stevelizcano** [fleet-ai/context#1]
Usage:

```sh
❯ context
Welcome to Fleet Context!
Generate and run code using the most up-to-date libraries.


> how to do fastapi request logging?



Traceback (most recent call last):
  File "/Users/stephenlizcano/miniconda3/lib/python3.11/site-packages/utils/ai.py", line 154, in get_openai_chat_response
    response = openai.ChatCompletion.create(
               ^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'openai' has no attribute 'ChatCompletion'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/stephenlizcano/miniconda3/bin/context", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/stephenlizcano/miniconda3/lib/python3.11/site-packages/cli.py", line 142, in main
    for response in get_openai_chat_response(prompts, model=model):
  File "/Users/stephenlizcano/miniconda3/lib/python3.11/site-packages/utils/ai.py", line 162, in get_openai_chat_response
    except openai.error.AuthenticationError as error:
           ^^^^^^^^^^^^
AttributeError: module 'openai' has no attribute 'error'
```

Installed on python 3.11, also have latest openai library installed.

**Comment by adrienB134** [comment]
Had the same issue, it seems that the "ChatCompletion" method is deprecated in the latest version of the openai library. 
You can get context to work by reverting to an older version of the openai lib:
=> pip uninstall openai
=> pip install "openai<1.0.0"

**Comment by adrwz** [comment]
Fixed! Now requiring openai>1.1.0 and using the new OpenAI Python API. [PR here](https://github.com/fleet-ai/context/pull/7). Feel free to use through `pip install --upgrade fleet-context`
